src/ProjectOrigin.Electricity.Example/WithoutWalletFlow.cs

```diff
@@ -76,7 +76,7 @@ public async Task<int> Run()
 
 
         // ------------------  allocate ------------------
-        Console.WriteLine($"Allocating Production Granular Certificate");
+        Console.WriteLine($"Creating allocation event");
```

Comment:
Why this change?

Reply:
originally a typo, same log statement few lines later..